storybook/src/components/MegaMenu/MegaMenu.stories.tsx

```diff
@@ -28,47 +25,44 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {
   args: {
-    children: (
-      <Grid key="nav" paddingVertical="medium">
-        <Grid.Cell span="all">
-          <Heading level={1} size="level-2">
-            Alle onderwerpen
-          </Heading>
-          <MegaMenu.ListCategory>
-            <LinkList>
-              <LinkList.Link href="#">Afval</LinkList.Link>
-              <LinkList.Link href="#">Bestuur en organisatie</LinkList.Link>
-              <LinkList.Link href="#">Bouw- en verkeersprojecten</LinkList.Link>
-              <LinkList.Link href="#">Burgerzaken</LinkList.Link>
-              <LinkList.Link href="#">Diversiteit</LinkList.Link>
-              <LinkList.Link href="#">Gemeentebelastingen</LinkList.Link>
-              <LinkList.Link href="#">Gezondheidsdienst (GGD)</LinkList.Link>
-              <LinkList.Link href="#">Kunst en cultuur</LinkList.Link>
-              <LinkList.Link href="#">Ondernemen</LinkList.Link>
-              <LinkList.Link href="#">Onderwijs en jeugd</LinkList.Link>
-              <LinkList.Link href="#">Parkeren</LinkList.Link>
-              <LinkList.Link href="#">Sport</LinkList.Link>
-              <LinkList.Link href="#">Stadsdelen</LinkList.Link>
-              <LinkList.Link href="#">Subsidies</LinkList.Link>
-              <LinkList.Link href="#">Vacatures</LinkList.Link>
-              <LinkList.Link href="#">Verkeer en vervoer</LinkList.Link>
-              <LinkList.Link href="#">Vrije tijd</LinkList.Link>
-              <LinkList.Link href="#">Werk aan de weg</LinkList.Link>
-              <LinkList.Link href="#">Werk en inkomen</LinkList.Link>
-              <LinkList.Link href="#">Wonen en leefomgeving</LinkList.Link>
-            </LinkList>
-          </MegaMenu.ListCategory>
-        </Grid.Cell>
-      </Grid>
-    ),
+    children: [
+      <Heading level={1} size="level-2">
+        Alle onderwerpen
+      </Heading>,
+      <MegaMenu.ListCategory>
```

Comment:
MegaMenu is deprecated??

Reply:
Yes. It has never done much; see the low amount of tokens and CSS. The outer `nav` will now be rendered through a `Grid`; its only remaining feature is the CSS columns, for which we can make an independent `Columns` component.